test/system/version.bats

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bats
+
+load 'libs/bats-support/load'
+load 'libs/bats-assert/load'
+
+@test "Output version number using full flag" {
+  run toolbox --version
+
+  assert_output --regexp '^toolbox version [0-9]+\.[0-9]+\.[0-9]+$'
+}
+
+@test "Output version number using command" {
```

Comment:
Should the test case be named as it is when the command, clearly, is not implemented (I don't know if it ever will be)?

Reply:
I guess this doesn't make much sense. I was thinking about testing when a non-implemented command is passed, which should raise an error. Also, the possibility of that command be implemented.
**Podman** has both, the flag and the command, so I guessed **Toolbox** would implement it in the future.

BTW, the version functionality doesn't appear at the autocomplete nor at the help or man, so I didn't know for sure the expected behavior.

Anyway, I was thinking to convert the test case into a `non-valid-command` case. So, it assures that if a non-valid-command is used, it exits with code `1`, show a clear explanation and how to get the help.
Basically, what this test case does, but as a `non-valid-commnd` test, insted of `command version` test.
Sounds good?